src/Waypoint/WaypointReaderSeeYou.hpp

```diff
@@ -44,6 +44,11 @@ class WaypointReaderSeeYou final : public WaypointReaderBase {
 protected:
   /* virtual methods from class WaypointReaderBase */
   bool ParseLine(const TCHAR* line, Waypoints &way_points) override;
+
+private:
+  /* field positions for typical SeeYou *.cup waypoint file */
```

Comment:
Variables belong to the top, before the constructor, according to our coding style.